src/Tag/Tag.js

```diff
@@ -21,21 +22,28 @@ const tagToTextSize = {
 class Tag extends WixComponent {
   static displayName = 'Tag';
 
+  componentDidMount() {
+    if (this.props.hasOwnProperty('wrap'))
```

Comment:
I think that the following is enough:
```suggestion
    if (this.props.wrap)
```